src/classes/BDI_FieldLevelSecurityAccessLevel.cls

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
As we talked about having this in its own class feels excessive ... can we move it into a member of UTIL_Permissions or perhaps BDI_DataImportFLSService if it works with the inheritance model?